docs/layouts/documentation.css

```diff
@@ -5,22 +5,33 @@
 }
 
 .sidebar {
-  min-width: 320px;
+  min-width: 360px;
+  max-width: 360px;
```

Comment:
This reminds me of the code in @ghh's [other pr](https://github.com/dataform-co/dataform-co/pull/6085#discussion_r428730314) - looks like it can be `width: 360px` only. 

Might turn out I'm wrong though!